README.md

```diff
@@ -95,35 +96,33 @@ $ sudo mkdir -p -m0700 /var/lib/teleport
 $ sudo chown $USER /var/lib/teleport
 ```
 
-If the build succeeds the binaries will be placed in
+If the build succeeds, the installer places the binaries in the following directory:
 `$GOPATH/src/github.com/gravitational/teleport/build`
 
-NOTE: The Go compiler is somewhat sensitive to amount of memory: you will need
-at least 1GB of virtual memory to compile Teleport. 512MB instance without swap
-will not work.
-
-NOTE: This will build the latest version of Teleport, regardless of whether it is stable. If you want to build the latest stable release, `git checkout` to that tag (e.g. `git checkout v6.0.0`) before running `make full`.
+**Important:**
+* The Go compiler is somewhat sensitive to the amount of memory: you will need **at least** 1GB of virtual memory to compile Teleport. A 512MB instance without swap will **not** work.
+* This will build the latest **version** of Teleport, **regardless** of whether it is stable. If you want to build the latest stable release, run `git checkout` to the corresponding tag (for example, run `git checkout v6.0.0`) **before** running `make full`.
```

Comment:
```suggestion
* This will build the latest version of Teleport, **regardless** of whether it is stable. If you want to build the latest stable release, run `git checkout` to the corresponding tag (for example, run `git checkout v6.0.0`) **before** running `make full`.
```

Reply:
That's a good call, I like it!

Reply:
Ah, you know what happened? Silly mistake. I meant to bold the "latest" not "version". However, this update should be fine as is.

Reply:
Thanks for looking at this BTW!